lib/js/atoms/Select/Select.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+	<div class="a-select" :class="{ '-error': isErrorComputed, '-disabled': disabled }">
+		<ds-app>
+			<v-select
+				:items="['a', 'b', 'c']"
+				:messages="errorMessage || helpMessage"
+				:error="isErrorComputed"
+				:readonly="readonly"
+				:disabled="disabled"
+				filled
+			>
+				<template slot="message">
+					<div class="a-select__helpMessage">
+						<icon
+							v-if="isErrorComputed"
+							class="a-input__labelErrorIcon"
+							:icon="ICONS.FA_EXCLAMATION_TRIANGLE"
+							:size="ICON_SIZES.XX_SMALL"
+						></icon>
+						<span>
+							{{ errorMessage || helpMessage }}
+						</span>
+					</div>
+				</template>
+
+				<template #prepend-inner>
+					<template v-if="leftIcon">
+						<icon :icon="leftIcon" :size="ICON_SIZES.X_SMALL"></icon>
+					</template>
+				</template>
+
+				<template #append>
+					<icon
```

Comment:
do we want rotate this icon when menu is open?

Reply:
I don't know it yet. In current implementation chevron is not visible when select is open